commons/ava_testsuite/conflicting_txs_vertex_test/conflicting_txs_vertex_test_.go

```diff
@@ -0,0 +1,223 @@
+package conflicting_txs_vertex_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ava-labs/gecko/snow/choices"
+
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_networks"
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_services"
+	"github.com/kurtosis-tech/kurtosis/commons/networks"
+	"github.com/kurtosis-tech/kurtosis/commons/testsuite"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	normalNodeConfigId            = 1
+	byzantineConfigId             = 2
+	byzantineUsername             = "byzantine_gecko"
+	byzantinePassword             = "byzant1n3!"
+	byzantineConflictingTxsVertex = "conflicting-txs-vertex"
+	stakerUsername                = "staker_gecko"
+	stakerPassword                = "test34test!23"
+	byzantineNodeServiceId        = 0
+	normalNodeServiceId           = 1
+	seedAmount                    = int64(50000000000000)
+	stakeAmount                   = int64(30000000000000)
+)
+
+// ================ Byzantine Test - Conflicting Transactions in a Vertex Test ===================================
+// StakingNetworkConflictingTxsVertexTest implements the Test interface
+type StakingNetworkConflictingTxsVertexTest struct {
+	ByzantineImageName string
+	NormalImageName    string
+}
+
+// Issue conflicting transactions to the byzantine node to be issued into a vertex
+// The byzantine node should mark them as accepted when it issues them into a vertex.
+// Once the transactions are issued, verify the byzantine node has marked them as accepted
+// Virtuous nodes should drop the vertex without issuing it the vertex or its transactions
+// into consensus.
+// As a result both the virtuous and rogue transactions within the vertex should stay stuck
+// in processing.
+func (test StakingNetworkConflictingTxsVertexTest) Run(network networks.Network, context testsuite.TestContext) {
+	castedNetwork := network.(ava_networks.TestGeckoNetwork)
+	logrus.Info("Changes are being reflected")
```

Comment:
Was this a leftover log statement from when changes to the file weren't getting incorporated into the Docker build?

Reply:
yes, removing it